README.zh-CN.md

```diff
@@ -4,7 +4,7 @@
       <img width="100%" src="https://raw.githubusercontent.com/ultralytics/assets/main/yolov5/v70/splash.png"></a>
   </p>
 
-[英文](README.md)|[简体中文](README.zh-CN.md)<br>
+[英文](YOLOv5_README.md)|[简体中文](README.zh-CN.md)<br>
```

Comment:
Good morning!! What is this for?

Reply:
this is their READMEs, I did not want to remove them, but I would like our readme to appear on the front page of the repo, with a link to the ultralytics readme.